ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py

```diff
@@ -0,0 +1,34 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# asanyarraya
+
+@handle_frontend_test(
+    fn_tree="numpy.asanyarray",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("valid")),
+)
+def test_numpy_asanyarray(
+    *,
+    dtype_and_x,
+    as_variable,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        a=x,
+        dtype=input_dtype[0],
```

Comment:
Missing some of the keyword arguments of `asanyarray` as input to `test_frontend_function`

Reply:
Do you mean the above?

Reply:
Currently there are just 2 arguments from the `asanyarray` included in the test: `a=x, dtype=input_dtype[0]`, but there are two more keyword arguments defined in the `asanyarray` call signature.